cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs

```diff
@@ -179,7 +183,11 @@ genesisBenchmarkRunner loggingLayer
   when (tps < 0.05) $
     left $ TooSmallTPSRate tps
 
-  let (benchTracer, connectTracer, submitTracer, lowLevelSubmitTracer) = createTracers loggingLayer
+  let ( benchTracer
+       , connectTracer
+       , submitMuxTracer
```

Comment:
alignment off

Reply:
It's not off -- it's a parse error if I align `,` with `(`.

Reply:
then it might be better to put the comma at the end of the line and align the names?